content/app/development/ux/fields/grouping/non-repeating/_index.en.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Grouping of fields
+linktitle: Non-repeating
+description: Setup for non-repeating groups
+weight: 2
+---
+
+Fields in a form can be set up to be part of a _group_. This can be used to e.g. set up dynamic on multiple
```

Comment:
Did you mean dynamics?

```suggestion
Fields in a form can be set up to be part of a _group_. This can be used to e.g. set up dynamics on multiple
```